package.json

```diff
@@ -214,7 +214,8 @@
     "fix": "yarn fix:other && yarn fix:code",
     "fix:other": "yarn prettier --write && yarn stylelint --fix",
     "fix:code": "yarn lint:code --fix",
-    "lint": "yarn lint:other && yarn lint:code",
+    "lint": "yarn lint:other && yarn lint:ts && yarn lint:code",
+    "lint:ts": "tsc --project tsconfig.build.json --noEmit",
```

Comment:
This will slowdown the `lint` job and the typecheck is already done in the building step. So we would do it twice

Reply:
fair, its gone